source/libNsdl/src/sn_grs.c

```diff
@@ -491,13 +482,12 @@ int8_t sn_grs_put_resource(struct grs_s *handle, sn_nsdl_dynamic_resource_parame
     }
 
     /* Check path validity */
-    if (!res->static_resource_parameters->pathlen || !res->static_resource_parameters->path) {
+    if (!res->static_resource_parameters->path) {
```

Comment:
Original version checked also that path's length is >0, should that functionality be maintained here? One could check that eg. by adding a (res->static_resource_parameters->path[0] == '\0') check.